### DISCUSSION TITLE
Bug: The L button is broken for GPRadio

### DISCUSSION THREAD

**Discussion by bartlangelaan** [f1multiviewer/issue-tracker#117] [Feature requests]
According to the documentation of GPRadio, you should be able to adjust the _targetLatency_ using the arrow up/down, and pressing L should then jump to live, taking _targetLatency_ into effect.

However, this seems to be broken. If the targetLatency is 20 seconds, I would expect that if I press L it plays the stream with a 20 sec delay. However, it moves the player to 20 seconds.

This was already [confirmed on Discord](https://discord.com/channels/982375740764983366/983777853055312002/995369139642310696) - so this is just to keep track of the bug.

Also, this is related but not a duplicate of https://github.com/f1multiviewer/issue-tracker/discussions/105.

**Comment by jeppeolesen** [comment]
Hey there @bartlangelaan 

If this is still an issue, please submit a but report here: https://github.com/f1multiviewer/issue-tracker/issues
